validation/report.md

```diff
@@ -4,23 +4,35 @@ This report summarizes the Phase II validation efforts by the global indicators
 
 ## Literature Review
 
-Over the past decade, the internet has transitioned from an emphasis on users consuming predefined content to users contributing and sharing content (O’Reilly 2005). When working with geodata, quality is often defined by pre-established parameters that have been compiled over the years. In the field of geo-information, the principles of the International Organization for Standardization are taken as a standard of quality assessment. The OpenStreetMap Project (OSM) emerged from this growing emphasis on user-generated content. OSM is a system based on volunteered geographic information (VGI), which is often questioned with regards to the quality of the content of the provided spatial data (Girres and Touya 2010). OSM has produced a large amount of spatial data. However, because most of its contributors are volunteers or aficionados from diverse backgrounds who may lack training in geographical information science standards and methods, quality has always been a concern (Singh Sehra et al. 2020). Numerous scientific studies have focused on evaluating OSM data quality. Over time, various approaches have been developed as technological advancements have provided new tools that have resulted in more complex quality assessment methods. 
+Over the past decade, the emphasis of the internet transitioned from users consuming predefined content to users simply contributing and sharing content (O’Reilly 2005). When working with geodata, quality is often defined by pre-established parameters compiled over the years. In the field of geo-information, the principles of the International Organization for Standardization are taken as a standard of quality assessment. 
```

Comment:
Standards are different from quality. ISO deals with the former. If they deal with the latter in some way, that should be made explicit and cited here. I'm not clear that they do.